htdp-lib/lang/private/teach.rkt

```diff
@@ -467,7 +467,8 @@
                             #%top
                             empty true false
                             beginner-quote
-                            intermediate-quote)))))
+                            intermediate-quote
+                            intermediate-quasiquote)))))
```

Comment:
Sorry, I should have been clearer. Quasiquote comes with unquote, "married."

Reply:
Unquote is already a keyword (in ISL).